app/scripts/modules/google/src/instance/gceInstanceType.service.js

```diff
@@ -317,20 +318,26 @@ module.exports = angular
       });
     }
 
-    function getAvailableTypesForLocations(instanceTypes, locationToInstanceTypesMap, selectedLocations) {
+    function getAvailableTypesForLocations(locationToInstanceTypesMap, selectedLocations) {
       // This function is only ever called with one location.
-      let [location] = selectedLocations,
-        availableTypesForLocation = locationToInstanceTypesMap[location].instanceTypes;
-
-      return _.intersection(instanceTypes, availableTypesForLocation);
+      let [location] = selectedLocations;
```

Comment:
use `const` instead of `let` here (I know it was a `let` before, just might as well clean it up while you're in there)